error[E0277]: `ContextError` must be implemented for `MyError`, or any error type captured by custom contexts
  --> tests/ui/context_error_hint_error.rs:15:45
   |
15 |     let _cx = context::new().with_capture::<MyError>();
   |                              ------------   ^^^^^^^ the trait `ContextError<musli::alloc::System>` is not implemented for `MyError`
   |                              |
   |                              required by a bound introduced by this call
   |
   = note: use `musli::context::ErrorMarker` to ignore errors
   = note: use `std::io::Error` and `std::string::String`, if the `std` or `alloc` features are enabled for `musli`
   = help: the following other types implement trait `ContextError<A>`:
             ErrorMarker
             musli::descriptive::Error<A>
             musli::json::Error<A>
             musli::packed::Error<A>
             musli::storage::Error<A>
             musli::value::Error<A>
             musli::wire::Error<A>
             std::io::Error
             std::string::String
note: required by a bound in `DefaultContext::<A, T, C>::with_capture`
  --> src/context/default_context.rs
   |
   |     pub fn with_capture<E>(self) -> DefaultContext<A, T, Capture<E>>
   |            ------------ required by a bound in this associated function
   |     where
   |         E: ContextError<A>,
   |            ^^^^^^^^^^^^^^^ required by this bound in `DefaultContext::<A, T, C>::with_capture`
